R/positron.R

```diff
@@ -0,0 +1,89 @@
+ark_register_methods <- function() {
+  # TODO: Actually bump this to 0.1.176 after Ark PR merges and we bump the version
+  if (!ark_has_version("0.1.174")) {
+    return()
+  }
+
+  tryCatch(
+    ark_register_methods_impl(),
+    error = function(err) {
+      cli::cli_warn("Failed to register Ark methods.", parent = err)
+    }
+  )
+}
+
+ark_register_methods_impl <- function() {
+  ark_register_method(
+    "ark_positron_variable_display_value",
+    "duckplyr_df",
+    function(x, ...) {
+      duckplyr_df_variable_display_value(x)
+    }
```

Comment:
```suggestion
    duckplyr_df_variable_display_value
```

with a suitable definition of `duckplyr_df_variable_display_value()` ?